src/NuGet/deprecated-templates-Install.ps1

```diff
@@ -0,0 +1,5 @@
+param($installPath, $toolsPath, $package, $project)
+
+write-host Removing this package since it is deprecated in favor of Microsoft.Orleans.OrleansCodeGenerator.Build
+
+uninstall-package $package.Id -ProjectName $project.Name
```

Comment:
This just uninstalls the dummy Microsoft.Orleans.Templates.Interfaces/Grains package right after it finished upgrading to 1.2.0. It could also uninstall the runtime codegen package which was previously included unnecessarily as a dependency, but I'm worried that somehow this will break someone's environment in a non trivial to troubleshoot way